console/src/components/Services/Data/TablePermissions/Permissions.js

```diff
@@ -54,20 +54,27 @@ class Permissions extends Component {
   constructor() {
     super();
     this.state = {};
-    this.state.insertSetOperations = {
-      isChecked: false,
-      columnTypeMap: {},
+    this.state.setOperations = {
+      insert: {
+        isChecked: false,
+        columnTypeMap: {},
+      },
+      update: {
+        isChecked: false,
+        columnTypeMap: {},
+      },
     };
     this.state.viewInfo = {};
     this.state.showAggregation = false;
     this.state.showInsertPrefix = false;
-    this.onSetValueBlur = this.onSetValueBlur.bind(this);
+    this.state.showUpdatePresets = false;
   }
   componentDidMount() {
     if (this.props.serverVersion) {
-      this.checkSemVer(this.props.serverVersion).then(() =>
-        this.checkPrefixVer(this.props.serverVersion)
-      );
+      this.checkSemVer(this.props.serverVersion).then(() => {
+        this.checkPrefixVer(this.props.serverVersion);
+        this.checkUpdatePresetVer(this.props.serverVersion);
```

Comment:
There could be a possibility where states gets overwritten. Both the functions are updating state. It would be nice if states are updated one after the other deterministically